lib/StripesFinalFormWrapper.js

```diff
@@ -121,7 +121,6 @@ class StripesFinalFormWrapper extends Component {
             <FormSpy
               subscription={{
                 dirty: true,
-                submitSucceeded: true,
```

Comment:
It's hard to predict how this change will affect all the other forms in FOLIO. It might be safer to add a separate prop for FormSpy configuration.